### DISCUSSION TITLE
Would the mod be updated to 1.20?

### DISCUSSION THREAD

**Discussion by Madesh-sudo** [LXYan2333/Fabric-Bedrock-Miner#64] [General]
Title i guess, wanted to ask whether the mod would be updated to 1.20

**Comment by bunnyi116** [comment]
I modified the version information in the original code from the mod author and fixed the methods that were removed in version 1.20. You can download it from this link: `https://github.com/Bunnui/Fabric-Bedrock-Miner/suites/13511611487/artifacts/742583454`, which was automatically generated from Github Actions. Alternatively, you can also try my modified branch at `https://github.com/Bunnui/Fabric-Bedrock-Miner/releases/tag/v1.20`

**Reply by ToiletKun** [reply]
oooooooooooof saving my life. much thanks

**Comment by LXYan2333** [comment]
Next month maybe
Now 1.20 is still in RC.

**Reply by bunnyi116** [reply]
> Next month maybe Now 1.20 is still in RC.

The official release of version 1.20 has been published.

**Comment by LXYan2333** [comment]
https://modrinth.com/mod/fabric-bedrock-miner for mc 1.20.1 and 1.20.6 avaliable
